runtime/runtime_test.go

```diff
@@ -4261,6 +4267,19 @@ func TestInterpretResourceOwnerFieldUseComposite(t *testing.T) {
 		log: func(message string) {
 			loggedMessages = append(loggedMessages, message)
 		},
+		getStorageUsed: func(_ Address) (uint64, error) {
+			var amount uint64 = 0
+
+			for _, data := range storage.storedValues {
+				amount += uint64(len(data))
+			}
+
+			return amount, nil
```

Comment:
This is a good mock implementation, but makes the use and assertions below (https://github.com/onflow/cadence/pull/770/files#diff-74ffa81b6938856f695ba112a721503dfc5ff2e852bc65d79d15ee4868f4bb97R4333) make this test dependent on the storage format, the size of the encoded data might change, breaking this test unnecessarily. 

Maybe just return a unique constant value like in `getStorageCapacity` below